docs/examples/irradiance-transposition/use_perez_modelchain.py

```diff
@@ -0,0 +1,124 @@
+"""
+Use different Perez coefficients with the ModelChain
+====================================================
+
+This example demonstrates how to customize the ModelChain
+to use site-specific Perez coefficients.
+"""
+
+# %%
+# The :py:class:`pvlib.modelchain.ModelChain` object provides a useful method
+# for easily constructing a system model with a simple, unified interface.
+# However, a user may want to customize the steps
+# in the system model in various ways.
+# One such example is during the irradiance transposition step.
+# The Perez models perform very well on field data, but
+# they require a set of fitted coefficients from various sites.
+# It has been noted that these coefficients can be specific to
+# various climates, so users may see improved model accuracy
+# when using a site-specific set of coefficients.
+# However, the base  :py:class:`~pvlib.modelchain.ModelChain`
+# only supports the default coefficients.
+# This example shows how the  :py:class:`~pvlib.modelchain.ModelChain` can
+# be adjusted to use a different set of Perez coefficients.
+
+import pandas as pd
+from pvlib.pvsystem import PVSystem
+from pvlib.modelchain import ModelChain
+from pvlib.temperature import TEMPERATURE_MODEL_PARAMETERS
+from pvlib import iotools, location, irradiance
+import pvlib
+import os
+import matplotlib.pyplot as plt
+
+# load in TMY weather data from North Carolina included with pvlib
+PVLIB_DIR = pvlib.__path__[0]
+DATA_FILE = os.path.join(PVLIB_DIR, 'data', '723170TYA.CSV')
+
+tmy, metadata = iotools.read_tmy3(DATA_FILE, coerce_year=1990,
+                                  map_variables=True)
+
+weather_data = pd.DataFrame({'ghi': tmy['ghi'], 'dhi': tmy['dhi'],
+                             'dni': tmy['dni'],
+                             'temp_air': tmy['temp_air'],
+                             'wind_speed': tmy['wind_speed']})
+
+loc = location.Location.from_tmy(metadata)
+
```

Comment:
I'm inclined to break the code block in two at this point, just to make it less overwhelming to users. This can be done by adding in a sentence saying something like "Next, we set up a standard PV model using ModelChain".

Reply:
Yeah, agreed, done